devices/lidl.js

```diff
@@ -155,6 +155,17 @@ module.exports = [
             device.getEndpoint(1).saveClusterAttributeKeyValue('lightingColorCtrl', {colorCapabilities: 29});
         },
     },
+    {
+        fingerprint: [{modelID: 'TS0505B', manufacturerName: '_TZ3000_qd7hej8u'}],
+        model: 'HG07834C',
+        vendor: 'Lidl',
+        description: 'Livarno Lux E27 bulb RGB',
+        ...extend.light_onoff_brightness_colortemp_color({disableColorTempStartup: true}),
```

Comment:
Please add the color temperature range as described on https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html#_3-adding-converter-s-for-your-device

Reply:
Added to the exception, just like HG07834C as they are very alike

Reply:
What results do you get when reading the min/max color temp? I prefer not to add new devices to the exceptions list.

Reply:
I got `Read result of 'lightingColorCtrl': {"colorTempPhysicalMin":153,"colorTempPhysicalMax":500}`

And I have it set in the code now as 

```
...extend.light_onoff_brightness_colortemp_color({disableColorTempStartup: true, colorTempRange: [153, 500]}),
```

But I'm wondering if I should keep the `disableColorTempStartup` . How can I know this?

Reply:
Please update the pr with this code, these bulbs do not support setting the color temp startup therefore it is disabled

Reply:
That's done! Thanks for the help!